tests/kafkatest/services/security/security_config.py

```diff
@@ -79,27 +87,54 @@ def __init__(self, security_protocol, template_props=""):
             security_protocol = self.get_property('security.protocol', template_props)
         if security_protocol is None:
             security_protocol = SecurityConfig.PLAINTEXT
-        elif security_protocol not in [SecurityConfig.PLAINTEXT, SecurityConfig.SSL]:
+        elif security_protocol not in [SecurityConfig.PLAINTEXT, SecurityConfig.SSL, SecurityConfig.SASL_PLAINTEXT, SecurityConfig.SASL_SSL]:
             raise Exception("Invalid security.protocol in template properties: " + security_protocol)
 
+        if interbroker_security_protocol is None:
+            interbroker_security_protocol = security_protocol
+        self.interbroker_security_protocol = interbroker_security_protocol
+        if sasl_mechanism is None:
```

Comment:
Can `sasl_mechanism` just default to `SASL_MECHANISM_GSSAPI` in signature of the constructor?